core/blockchain.go

```diff
@@ -2071,8 +2071,15 @@ func (bc *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 	for _, tx := range types.HashDifference(deletedTxs, addedTxs) {
 		rawdb.DeleteTxLookupEntry(indexesBatch, tx)
 	}
-	// Delete any canonical number assignments above the new head
-	number := bc.CurrentBlock().NumberU64()
+	deletedTxs, addedTxs = nil, nil // release the references in case the slices are huge
```

Comment:
Heh, the linter doesn't like this
```
core/blockchain.go:2074:2: ineffectual assignment to deletedTxs (ineffassign)
	deletedTxs, addedTxs = nil, nil // release the references in case the slices are huge
	^
util.go:47: exit status 1
```
Not sure what to do here.